interchaintest/client_threshold_test.go

```diff
@@ -164,15 +166,16 @@ func TestScenarioClientTrustingPeriodUpdate(t *testing.T) {
 	g0, g1 := chains[0], chains[1]
 
 	client, network := interchaintest.DockerSetup(t)
-	relayerinterchaintest.BuildRelayerImage(t)
+	uuid := relayerinterchaintest.UniqueRelayerImageName()
```

Comment:
nit: rename `uuid` to something that more clearly indicates what this variable is, e.g. `image`